### DISCUSSION TITLE
custom.javascript convert process

### DISCUSSION THREAD

**Discussion by ZsadanyiDani** [ConduitPlatform/Conduit#1194] [Q&A]
Hello guys, i would like to write a custom.javascript process to convert a string like this "2024-05-17T13:49:21.337Z" to a DATETIME(3) type date. I made a sql-server to mysql pipeline and the sql-server source connector makes a string from the original date type field.
I tried a lot of different function but nothing worked.

pipelines:
- id: marsbasepipeline
  status: running
  name: mars-base-pipeline
  description: desc
  connectors:
  - id: con1
    type: source
    plugin: standalone:sql-server
    name: my-sql-source
    settings:
      connection: sqlserver://user:pass@adress:port?database=dbname
      table: MARS_BASE
      orderingColumn: uuid
      columns: uuid,status,type,requestor,projectLeader,customer,customerProjectName,productSalesname,created,flowTemplateId,segment,security
  - id: con2
    type: destination
    plugin: standalone:mysql
    name: my-sql-destination
    settings:
      url: host:pass@tcp(localhost:3306)/mars_base_test
      table: mars_base_copy
      key: uuid
      
This is my pipeline yaml, really basic simple, and it works if i make my 'created' field what i need to convert a VARCHAR, but i would like it in DATETIME(3).

Can someone help me with this.

**Comment by kkopanidis** [comment]
Hey @ZsadanyiDani, I'm afraid you are in the wrong repo and we can't help you with your issue :/
